package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Dependent-Jaguar, MetroRail: Data Modeling & Database Design (LG Goal 126)",
   "main": "app.js",
   "scripts": {
-    "test": "mocha --watch dist/spec",
+    "test": "./node_modules/mocha/bin/_mocha",
```

Comment:
@deonna I spent a lot of time googling and looking at old projects to try and determine how to actually call my tests.  There's a lot of stuff going on in past projects that I don't understand but I never tried just directly using ' "test":"mocha" '!  That works now.  

Is there a different way I should be doing it, or is this good?